internal/pulsarutils/pulsar_to_channel.go

```diff
@@ -0,0 +1,35 @@
+package pulsarutils
+
+import (
+	"context"
+
+	"github.com/severinson/pulsar-client-go/pulsar"
+)
+
+// PulsarToChannel is a service for receiving messages from Pulsar and forwarding those on C.
```

Comment:
It's not entirely clear why we need this- the client already lets you supply a channel that messages can be published to?

Reply:
I'm not sure- I guess the question is whether it logs errors and reconnects as expected.   If it does, what else would we do with the error? Maybe prometheus metrics?